codegens/dart/package.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
The name of the folder for this dart-http. 
also for the name here in the package json should include its variant in the name.

Reply:
Done